build.xml

```diff
@@ -72,7 +72,7 @@
   <property name="jni.revision" value="1"/>
   <property name="jni.build" value="0"/> <!--${build.number}-->
   <property name="jni.version" value="${jni.major}.${jni.minor}.${jni.revision}"/>
-  <property name="jni.md5" value="1a6047467b59e8748f975e03016ce3d9"/>
+  <property name="jni.md5" value="4f72f2799dfee6008a386bc40afd7428"/>
```

Comment:
Updated the PR (bumped major version from 4 to 5).
I don't think I can rebuild the natives myself unless there's some fancy cross compilation option in JNA I don't know about. I only have Linux x64. How can I do this?